### ISSUE TITLE
Color issue 

### ISSUE THREAD

**Issue by junhyukk** [jmiller656/EDSR-Tensorflow#6]
Hello, @jmiller656  Thank you for your work! 
I trained the super resolution models (scale = 4) using General-100 and my own data, 
but the result image from both models have wrong color like this example..
![image](https://user-images.githubusercontent.com/31238236/30576468-0d6e4732-9d43-11e7-9590-fb18149b5523.png)

I am just wondering if it is caused by model structure or some trivial problem on your code made the image has wrong color.
Thanks.

**Comment by jmiller656** [comment]
I think this may be a result of the loss function that I've used (absolute difference). I don't remember if I ended up using this because of the paper, or by my own choice. Surely there are better loss functions than this though. I'll try experimenting and get back to you within the next week with my findings.

Thanks for pointing this out! I'm a bit colorblind and didn't quite notice at first.

**Comment by junhyukk** [comment]
The EDSR model from the paper used "absolute difference" loss function, so I think it is not the reason why the result was wrong. I tried to find the reason but I couldn't find yet..

If you find the reason, please let me know. Thanks for your answer.

**Comment by idearibosome** [comment]
I've created a pull request which is related to this issue. (#7)

Br,

**Comment by junhyukk** [comment]
Hi, @idearibosome 
Thank you very much! Through your advice, I solved the problem :)

**Comment by jmiller656** [comment]
Thanks, @idearibosome. I've just merged your pull request. I'll be closing this issue now.
